css/main2.css

```
@@ -238,4 +244,16 @@ table.conventions {
 table.conventions td {
     padding-left: 10px;
     padding-right: 10px;
+}
+
```

Suggested change:

```suggestion

table.links {
    width: 100%;
}

```

This, coupled with the corresponding change to the _Donations_ `td` width makes things look better, in my opinion.